### ISSUE TITLE
(V2) Build error: `XRSettings' does not exist in the namespace `UnityEngine'

### ISSUE THREAD

**Issue by silentlamb1991** [Unity-Technologies/PostProcessing#339]
##### What happened?
Assets/Plugins/PostProcessing/Runtime/Utils/RuntimeUtilities.cs(239,36): error CS0234: The type or namespace name `XRSettings' does not exist in the namespace `UnityEngine'. Are you missing an assembly reference?

##### Unity version, operating system, target platform (standalone windows, mac, iOS, PS4...)?
Unity 2017.2.0f3, standalone windows

**Comment by Chman** [comment]
I can't reproduce this issue. Are you sure your build target is set to windows and not another platform?

**Comment by silentlamb1991** [comment]
I just try again, error still exists.

![tt2](https://user-images.githubusercontent.com/18855621/32976729-82f7d146-cc58-11e7-84f6-d1b1cc3cf0cf.PNG)

![2](https://user-images.githubusercontent.com/18855621/32976718-2e8a247e-cc58-11e7-8348-a8698610d18e.PNG)

**Comment by silentlamb1991** [comment]
It seems fixed recent days.

**Comment by ruidazeng** [comment]
I am having this issue that says this when I tried to use setup my antilatency hardware. Normally it opens to version "2017.2.0" but when I imported package, the version seems to be changing to "5.5.2". I am new to Unity, is there a standard way to solve this?

`Assets/Antilatency/Integration.Oculus/Scripts/AltTrackingOculus.cs(6,19): error CS0234: The type or namespace name XRSettings' does not exist in the namespace UnityEngine'. Are you missing an assembly reference?`
